packages/core/helper-plugin/src/components/RelativeTime/index.js

```diff
@@ -4,7 +4,18 @@ import { intervalToDuration, isPast } from 'date-fns';
 import PropTypes from 'prop-types';
 import { useIntl } from 'react-intl';
 
-const RelativeTime = ({ timestamp }) => {
+/**
```

Comment:
Maybe this comment is not necessary if you create the story in https://github.com/strapi/strapi/blob/6530915e3d3fe173bd063b17230e7adbbe77e14d/packages/core/helper-plugin/src/components/RelativeTime/RelativeTime.stories.mdx

Reply:
Or both, but for me a comment is still important :)

Reply:
updated RelativeTime.stories.mdx and left the component jsdoc too! 🚀